manifests/osfamily/solaris.pp

```diff
@@ -37,61 +37,63 @@
       }
     }
     '11': {
-      class { 'puppet_agent::prepare::package':
```

Comment:
That's a good question, I didn't add the bool there as there was no repo being created for Solaris 10...I guess it's possible that there could be externally managed repos for Solaris 10, so that block should have enable/disable functionality. Thoughts?